src/components/AssetSelection/components/AssetChainDropdown/AssetChainDropdown.tsx

```diff
@@ -77,11 +87,11 @@ export const AssetChainDropdown: React.FC<AssetChainDropdownProps> = memo(
       [onChangeAsset],
     )
 
-    const isDisabled = useMemo(() => {
-      return filteredRelatedAssetIds.length <= 1 || isLoading || isError
-    }, [filteredRelatedAssetIds, isError, isLoading])
+    const isButtonDisabled = useMemo(() => {
+      return isDisabled || filteredRelatedAssetIds.length <= 1 || isLoading || isError
```

Comment:
The actual fix for disabling related asset selection when asset selection is read only